simulation/traffic_simulator/include/traffic_simulator/api/api.hpp

```diff
@@ -331,6 +331,7 @@ class API
   FORWARD_TO_ENTITY_MANAGER(getRelativePose);
   FORWARD_TO_ENTITY_MANAGER(getStandStillDuration);
   FORWARD_TO_ENTITY_MANAGER(getTraveledDistance);
+  FORWARD_TO_ENTITY_MANAGER(getDefaultMatchingDistanceForLaneletPoseCalculation);
   FORWARD_TO_ENTITY_MANAGER(getV2ITrafficLight);
   FORWARD_TO_ENTITY_MANAGER(getV2ITrafficLights);
   FORWARD_TO_ENTITY_MANAGER(isEgoSpawned);
```

Comment:
I think this function should not used by API users.
Please move it to private function.

Reply:
It is a very small change, so it is unnecessary to retry Autoware Evaluator.